commands/operator-sdk/cmd/build.go

```diff
@@ -173,6 +180,18 @@ func buildFunc(cmd *cobra.Command, args []string) {
 	fmt.Fprintln(os.Stdout, string(o))
 
 	if enableTests {
+		buildTestCmd := exec.Command("go", "test", "-c", "-o", filepath.Join(wd, "tmp/_output/bin", filepath.Base(wd)+"-test"), testLocationBuild+"/...")
+		buildTestCmd.Env = goBuildEnv
+		o, err := buildTestCmd.CombinedOutput()
+		if err != nil {
+			cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("failed to build: (%v)", string(o)))
```

Comment:
Change this to:
```Go
log.Fatalf("failed to build: %v (%v)", err, string(o)))
```
And on Line 161 as well. We're phasing out cmdError.ExitWithError since that doesn't really add much.